src/qibolab/parameters.py

```diff
@@ -50,6 +54,38 @@ def fill(self, options: ExecutionParameters):
         return options
 
 
+class TwoQubitContainer(dict[QubitPairId, TwoQubitNatives]):
```

Comment:
Something else I noticed about the pairs and symmetry using the dummy, is that for `(2, 3)` we register both CZ and CNOT but because the CNOT is not symmetric, it correctly does not register `(3, 2)`. In principle, we could still register just the CZ, which is symmetric, for `(3, 2)`. However, this case is a bit tricky and I don't even think it is handled in 0.1, so I would certainly not do anything about it here.

Reply:
But, if you wish, you could open an issue about that

Reply:
#991 without milestone for now, as I don't consider it that much urgent.